hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/RequestAuditor.java

```diff
@@ -53,6 +55,14 @@ AuditMessage buildAuditMessage(AuditAction op,
    */
   Map<String, String> buildVolumeAuditMap(String volume);
 
+  /**
+   * Build auditMap with specified Delegation Token.
+   *
+   * @param token Delegation token.
+   * @return auditMap.
+   */
+  Map<String, String> buildTokenAuditMap(Token<OzoneTokenIdentifier> token);
```

Comment:
I think we should avoid adding this method to the `RequestAuditor` interface and to the base `OMClientRequest` base class.  It only applies to a small subset of request types.  The implementation does not use any state of the request, only formats the token given in parameter, so it can be a static utility method.